alaoweb/2020.alaoweb.org
# Awards

This PR:
- adds the encrypted awardee files to the repository
- displays `no_image.png`, if no image file to defined
- creates a `awards-base64.json.template` file to contain all of the ALAO awards, as the completed and encrypted `awards-base64.json` only contains that awards that were actually awarded in a given year.

To Test:
- set `showAwardWinners: true`
- ensure that awardee information appears correctly in `http://127.0.0.1:4000/awards/`

Closes #274 & #255
